### PR TITLE
[skip ci] MEP template

### PR DESCRIPTION
**Pull request by neza2017**
issue: #6365
Signed-off-by: yefu.chen <yefu.chen@zilliz.com>

### CODE REVIEW

File: docs/design_docs/MEP-Template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# MEP(Milvus Enhancement Proposals) Templates
+Current state: [One of "Under Discussion", "Accepted", "Rejected"]
+
+ISSUE: like to the github issue
```
Comment (xige-16) [inline docs/design_docs/MEP-Template.md:4]: like?

Comment (neza2017) [reply docs/design_docs/MEP-Template.md]: link

**Comment by xiaofan-luan** [comment]
also we may want to add some keywords for search.

such as performance enhance, load balance, delete,datanode, datacoord...

**Comment by xiaofan-luan** [comment]
ship it
/approve

**Pull request merged**